### PR TITLE
feat: add --drivers to load particular drivers

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Currently `appium` loads alls installed drivers. It increases the initial load time.
`--drivers` argument allows users to load only particular drivers. It will help to reduce such initial load time.

```
% node . --drivers=uiautomator2                            
[Appium] Welcome to Appium v2.0.0-beta.12 (REV cc8ea3bb458f2326f8e9871bad43af036a3f54a0)
[Appium] Non-default server args:
[Appium]   drivers: {
[Appium]     0: uiautomator2
[Appium]   }
[Appium]   tmpDir: /var/folders/y6/524wp8fx0xj5q1rf6fktjrb00000gn/T
[Appium] Attempting to load driver uiautomator2...
[Appium] Appium REST http interface listener started on 0.0.0.0:4723
[Appium] Available drivers:
[Appium]   - xcuitest@3.42.0 (automationName 'XCUITest')
[Appium]   - uiautomator2@1.65.1 (automationName 'UiAutomator2')
[Appium] Available plugins:
[Appium]   - fake@0.2.0
[Appium] No plugins activated. Use the --plugins flag with names of plugins to activate
```

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/cli/args.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { DEFAULT_BASE_PATH } from 'appium-base-driver';
-import { parseSecurityFeatures, parseDefaultCaps, parsePluginNames, parseInstallTypes } from './parser-helpers';
+import { parseSecurityFeatures, parseDefaultCaps, parsePluginNames, parseInstallTypes, parseDriverNames } from './parser-helpers';
```
Comment (mykola-mokhnach) [inline lib/cli/args.js:2]: this string looks too long

File: lib/cli/args.js [hunk 2]
```diff
@@ -34,6 +34,15 @@ const serverArgs = [
     dest: 'shell',
   }],
 
+  [['--drivers'], {
+    required: false,
+    default: [],
+    help: `A comma-separated list of installed drivers names that should be active for this ` +
```
Comment (mykola-mokhnach) [inline lib/cli/args.js:40]: driver names

**Comment by mykola-mokhnach** [comment]
should the documentation be updated somewhere?

**Review by jlipps — APPROVED** [review]
yes great idea! @mykola-mokhnach driver docs are still pending since i wanted the interface to be stable before writing docs. but i will add that soon.

**Pull request merged**